CHANGELOG.md

```diff
@@ -1,6 +1,94 @@
 # Unreleased
 
-Nothing Yet!
+This release was probably going to be several releases, but everything got finished at the same time, so here's a Mega Release!
+
+The headline features are:
+
+* New Support For Axo Releases, As An Alternative To Github Releases (Launching Soon™)
+* New Support For Generic Build Steps, Using Things Like Makefiles (Experimental)
```

Comment:
You don't actually need makefiles, that's just what the C example uses. It supports arbitrary buildsystems.

```suggestion
* New Support For Generic Build Steps, In Any Language (Experimental)
```